adapter.go

```diff
@@ -49,21 +59,31 @@ func finalizer(a *adapter) {
 
 // NewAdapter is the constructor for Adapter. If database name is not provided
 // in the Mongo URL, 'casbin' will be used as database name.
-func NewAdapter(url string) persist.Adapter {
-	dI, err := mgo.ParseURL(url)
-	if err != nil {
-		panic(err)
+func NewAdapter(url string, timeout ...interface{}) persist.Adapter {
+	if !strings.HasPrefix(url, "mongodb+srv://") && !strings.HasPrefix(url, "mongodb://") {
```

Comment:
why?

Reply:
mongo-driver requires that url must start with `mongodb`, it is not compatible with previous usage

I want it to continue using URLs like this `127.0.0.1:27017` but `mongodb://127.0.0.1:27017`

Reply:
Good idea.